src/main/java/de/tum/in/www1/artemis/domain/enumeration/DefaultChannelType.java

```diff
@@ -0,0 +1,16 @@
+package de.tum.in.www1.artemis.domain.enumeration;
+
+public enum DefaultChannelType {
+
+    ANNOUNCEMENT("announcement"), ORGANIZATION("organization"), RANDOM("random"), TECH_SUPPORT("tech-support"),;
+
+    private final String name;
+
+    DefaultChannelType(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
```

Comment:
Optional:
You could simply take the name of the num value, convert it to lower case and replace _ by -.